src/pages/email-change-request/[token].tsx

```diff
@@ -33,15 +34,25 @@ const EmailChangeRequest: React.FunctionComponent = () => {
                 <button
                   className="text-white bg-red-500 border-0 py-2 px-8 focus:outline-none hover:bg-red-600 rounded"
                   onClick={async () => {
-                    const {
-                      success,
-                      new_email,
-                    } = await confirmEmailChangeRequest(token)
+                    try {
+                      const {
+                        success,
+                        new_email,
+                      } = await confirmEmailChangeRequest(token)
 
-                    if (success === true) {
-                      setViewerEmail(new_email)
+                      if (success === true) {
+                        setViewerEmail(new_email)
 
-                      router.replace('/user')
+                        toast.success(
+                          "You've successfully updated your email address",
+                        )
+                        router.replace('/user')
+                      }
+                    } catch (e) {
+                      toast.error(
+                        'This link for changing your email has been used or has expired. Feel free to request a new link.',
+                        {duration: 6000, icon: '❌'},
+                      )
```

Comment:
Introduce success and error toasts for Email Change Confirmation to make it clearer that it succeeded or failed.